tests/rt_rrtmgp.conf

```diff
@@ -0,0 +1,37 @@
+
```

Comment:
Is this file just for debugging rrtmgp tests?

Reply:
See https://github.com/ufs-community/ufs-weather-model/pull/201#issuecomment-691176613 - yes.